ports/onnxruntime/project-config-template.cmake

```
@@ -0,0 +1,50 @@
+@PACKAGE_INIT@
+
+include(CMakeFindDependencyMacro)
+
+# sync with portfile.cmake and onnxruntime_vcpkg_deps.cmake ...
+find_dependency(absl REQUIRED)
```

General in this file: Never use `REQUIRED` with `find_dependency`. It prevents this macro from doing the right thing.
```suggestion
find_dependency(absl)
```

---

Removed in https://github.com/microsoft/vcpkg/pull/36850/commits/750223d667f26baae7bf6d27da0f4b784c2ac97c